crux-core/src/crux/cli.clj

```diff
@@ -68,6 +68,7 @@
     shutdown?))
 
 (defn start-node-from-command-line [args]
+  (cio/try-set-malloc-arena-max 4)
```

Comment:
Not all paths to Crux nodes come through this function. One for `crux.node/start-node`, or maybe even a top-level side effect?

Reply:
The idea is that this is mainly interesting in the case where someone starts up the actual app via command line, but we want to avoid them having to set this environment variable, doing this in the node feels a bit too intrusive.

As mentioned in the other comment I'm not sure. Like this tries to nudge Crux to be a bit better behaved in the general case.